en-US/Characters_Codex/06_Spellcasting/Spell_Descriptions/B/Blight.md

```diff
@@ -9,6 +9,7 @@ _4th-level necromancy_
 **Duration:** Instantaneous
 
 Necrotic energies deal `8d8` necrotic damage to the target.
+The target must make a Constitution saving throw. The target takes 8d8 necrotic damage on a failed save, or half as much damage on a successful one.
```

Comment:
You left in the damage part before the save. 😛 
How about this?
```markdown
...
**Duration:** Instantaneous

The target must make a Constitution saving throw.
It takes `8d8` necrotic damage on a failed save, or half as much damage on a successful one.
Undead and constructs are immune to this spell.
...
```
We don't have to specify "The target" twice.